project/build.scala

```diff
@@ -5,17 +5,22 @@ import scala.util.Properties
 // sbt-assembly
```

Comment:
This may not be the place to do it, but it would be great to replace this dated `build.scala` with a simpler `build.sbt`.